rocketpy/simulation/flight.py

```diff
@@ -2058,37 +2007,44 @@ def get_solution_at_time(self, t, atol=1e-3):
     # Transform solution array into Functions
     @funcify_method("Time (s)", "X (m)", "spline", "constant")
     def x(self):
-        """Rocket x position as a Function of time."""
+        """Rocket x position relative to the launch pad as a Function of
+        time."""
         return self.solution_array[:, [0, 1]]
 
     @funcify_method("Time (s)", "Y (m)", "spline", "constant")
     def y(self):
-        """Rocket y position as a Function of time."""
+        """Rocket y position relative to the lauch pad as a Function of
+        time."""
         return self.solution_array[:, [0, 2]]
 
     @funcify_method("Time (s)", "Z (m)", "spline", "constant")
     def z(self):
-        """Rocket z position as a Function of time."""
+        """Rocket z position relative to the launch pad as a Function of
+        time."""
         return self.solution_array[:, [0, 3]]
 
     @funcify_method("Time (s)", "Altitude AGL (m)", "spline", "constant")
     def altitude(self):
```

Comment:
in the future the `env.elevation` parameter should be a function of x and y, or even a function of lat and lon